backend/create_admin.py

```
@@ -19,20 +19,21 @@
-        admin.set_password('admin123')
-        db.session.add(admin)
+        from utils.admin_init import create_secure_admin
+        success, message, password = create_secure_admin()
+        if success:
+            print(f"SUCCESS: {message}")
+            if password:
+                print(f"IMPORTANT: Generated admin password: {password}")
+                print("Please save this password securely!")
+        else:
+            print(f"ERROR: {message}")
+            return
```

Ruff (0.11.9)

`return` statement outside of a function/method

(F706)